module/x/gravity/types/genesis.go

```diff
@@ -33,7 +33,7 @@ var (
 	ParamsStoreKeyBridgeContractChainID = []byte("BridgeChainID")
 
 	// ParamsStoreKeySignedSignerSetTxsWindow stores the signed blocks window
-	ParamsStoreKeySignedSignerSetTxsWindow = []byte("SignedSignerSetTxWindow")
```

Comment:
here the param key is slightly different from its name

Reply:
it is a breaking change and would require migration right?

Not sure if it is worth it considering we have already a running testet (for mainnet, may be okay)

Reply:
yeah, this is a breaking change and needs migration.
It seems no worth a migration so I'll restore this